app/models/race.rb

```diff
@@ -0,0 +1,148 @@
+# frozen_string_literal: true
+
```

Comment:
Consider a top level comment that this is part of an intended feature and not the current source of truth for race data.